pgjdbc/src/test/java/org/postgresql/test/jdbc4/StringTypeUnspecifiedArrayTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2007, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql.test.jdbc4;
+
+import org.postgresql.PGProperty;
+import org.postgresql.geometric.PGbox;
+import org.postgresql.test.jdbc2.BaseTest4;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.sql.Array;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Properties;
+
+@RunWith(Parameterized.class)
+public class StringTypeUnspecifiedArrayTest extends BaseTest4 {
```

Comment:
Please add the test in question to the relevant test suite

Reply:
@vlsi sure thing, ummm, what is the relevant test suite?

Reply:
`org.postgresql.test.jdbc2.Jdbc2TestSuite`